list.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="main.css">
```

Comment:
If we put a leading `/` before `main.css` we'll be sure it can be found no mater where the rendered file ends up living.